src/common/Utilities/TaskScheduler.cpp

```diff
@@ -0,0 +1,230 @@
+/*
+ * This file is part of the WarheadCore Project. See AUTHORS file for Copyright information
```

Comment:
please adapt this to AzerothCore, it's ok to mention WarheadCore as well but use the existing format